components/automate-ui/src/app/pages/data-feed-details/data-feed-details.component.ts

```diff
@@ -72,10 +72,8 @@ export class DataFeedDetailsComponent implements OnInit, OnDestroy {
     this.updateForm = this.fb.group({
       // Must stay in sync with error checks in data-feed-details.component.html
       name: ['', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]],
-      url: ['', [Validators.required,
-        Validators.pattern(Regex.patterns.NON_BLANK),
-        Validators.pattern(Regex.patterns.VALID_FQDN)
-      ]]
+      // Note that URL here may be FQDN -or- IP!
+      url: ['', [Validators.required, Validators.pattern(Regex.patterns.NON_BLANK)]]
```

Comment:
I'm thinking we don't want to kill all this validation for the future.  I believe the only thing that needs to be done here is to remove `Validators.pattern(Regex.patterns.NON_BLANK)`

Reply:
i'm gonna take a quick stab at this for you to see if it's all that is needed

Reply:
oh sorry!  you already got the one I was looking at - please disregard - not the non blank 🙏